core/licenses/jna-4.4.0.jar.sha1

```diff
@@ -1 +1 @@
-6edc9b4514969d768039acf43f04210b15658cd7
\ No newline at end of file
+d85e9ceee4d97e738524a231687ff28446376c97
```

Comment:
Right now we differentiate based on the group ID, I don't think we need more than that as this need release out of sync should be infrequent as you say indeed.